### PR TITLE
check dssp version before running DSSP

### PR DESCRIPTION
**Pull request by fgrunewald**
Quick check to see if DSSP has the correct version.

### CODE REVIEW

File: vermouth/dssp/dssp.py [hunk 1]
```diff
@@ -191,6 +193,12 @@ def run_dssp(system, executable='dssp', savefile=None, defer_writing=True):
     read_dssp2
         Parse a DSSP output.
     """
+    # check version
+    process = subprocess.run(["dssp", "--version"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    stdout_UTF8 = process.stdout.decode('UTF8')
```
Comment (pckroon) [inline vermouth/dssp/dssp.py:198]:
```suggestion
    version_found = process.stdout.decode('UTF8')
```

**Review by pckroon — APPROVED** [review]
Do we want to parse the version with e.g. `pkg_resources.packaging.version.parse` before comparing?
Optional though.

**Comment by fgrunewald** [comment]
@pckroon I thought looking for "version 3.0.0" in the output string is a bit safer cuz who knows if DSSP will format their output appropriately. Since this is a quick-fix I think it's not needed but as you like

File: vermouth/dssp/dssp.py [hunk 2]
```diff
@@ -191,6 +193,12 @@ def run_dssp(system, executable='dssp', savefile=None, defer_writing=True):
     read_dssp2
         Parse a DSSP output.
     """
+    # check version
+    process = subprocess.run(["dssp", "--version"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    version_found = process.stdout.decode('UTF8')
+    if version not in stdout_UTF8:
```
Comment (pckroon) [inline vermouth/dssp/dssp.py:199]:
```suggestion
    if version not in version_found:
```

**Review by pckroon — APPROVED** [review]
Good point.

**Review by pckroon — APPROVED** [review]

**Pull request merged**